src/config/packages/security.yaml

```diff
@@ -10,6 +10,9 @@ security:
             entity:
                 class: 'App\Entity\User'
                 property: 'email'
+    role_hierarchy:
```

Comment:
Nop bad roles!

check this out : Des profils utilisateurs (utilisateurs, vendeurs qui peuvent aussi être acheteurs et admins)